solarwinds_apm/configurator.py

```diff
@@ -559,19 +573,30 @@ def _add_all_instrumented_python_framework_versions(
 
             # Set up Instrumented Library Versions KVs with several special cases
             entry_point_name = entry_point.name
-            # Some OTel instrumentation libraries are named not exactly
-            # the same as the instrumented libraries!
+            # Some OTel instrumentation library entry point names are not exactly
+            # the same as their corresponding instrumented libraries
             # https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation/README.md
-            if entry_point_name == "aiohttp_client":
+            if entry_point_name == "aiohttp-client":
```

Comment:
The instrumentor is named [opentelemetry-instrumentation-aiohttp-client](https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation/opentelemetry-instrumentation-aiohttp-client)...